### ISSUE TITLE
Instructor Question

### ISSUE THREAD

**Issue by jshearon** [jshearon/bear-watcher#16]
I think I understand modules and how to use them. Could you guys take a look at how I've divided up my helper/data and /components files and tell me if I've divided up the code the "right" way? I tried to just include basic getter/setter functions in the data files and more complex functionality in  the components.

**Comment by luketlancaster** [comment]
yeah this all looks good. So the general "rule" we'll be following is all the UI goes in components. So any interaction with the dom (printing to it, getting values from it, handling events, etc) goes there. Any interaction with data goes in your data files, which you've done. So it's less about complexity, and more about responsibility.

Also, just as an fyi, for your functions like [this one](https://github.com/jshearon/bear-watcher/blob/master/javascripts/helpers/data/bears.js#L31-L39)

you can simplify it a bit using the "new" ES6 hotness:
```js
const setBears = (name, image) => {
  const id = Math.max(...bears.map(x => x.id)) + 1;
  const newBear = { 
    id,
    name,
    image,
  };
  bears.push(newBear);
}
```

so if your key names are the same as variable names, you don't have to repeat them (i.e. `name: name` becomes just `name` in the object definition)
